quadriga/src/main/webapp/WEB-INF/views/auth/editCollection.html

```diff
@@ -0,0 +1,27 @@
+<html layout:decorate="~{layouts/main}">
+<head>
+<title>Quadriga 2.0</title>
+</head>
+<body>
+	<div layout:fragment="content">
+		<span th:text="${alert_msg}"></span>
+		<h2 th:object="${collectionForm}">
+			Edit Collection - <span th:text="*{name}"></span>
+		</h2>
+
+		<form action="#" th:action="@{/auth/collections/{id}/edit(id=*{id})}"
+			method="POST" th:object="${collectionForm}">
+			<label>Collection Name: </label> <input th:field="*{name}"
+				class="form-control input-sm"></input> <span
+				th:if="${#fields.hasErrors('name')}" class="error">Please
+				enter a valid Collection name.</span> <br> <label>Description:</label>
```

Comment:
let's fix the indentation here (one line per tag for example)

Reply:
Also, the error message is not useful if the user doesn't know what a valid collection name is. So, either show the validation message that was actually thrown (which would be preferable as we can localize that if necessary) or provide information what a valid collection name is.

Reply:
The validation rule on the name field is '@Size(min=1)', so entering a blank name displays the default error message "size must be between 1 and 2147483647"
But if I change it to @NotBlank, which I think is more readable,  the default error message says "must not be blank"

So should I change the Validation rule in CollectionForm to @NotBlank and then just display the default error message?

Reply:
Yes and yes